docs/user/who-is.rst

```diff
@@ -0,0 +1,70 @@
+WHOIS Lookup
+============
+
+.. important::
+
+    The **WHOIS Lookup** feature is disabled by default.
+
+    To enable this feature, follow the `setup instructions
+    <controller_setup_whois_lookup_>`_ below, then activate it via the
+    :ref:`global setting or from the admin interface
+    <OPENWISP_CONTROLLER_WHOIS_ENABLED>`.
+
+.. warning::
+
+    If :ref:`OPENWISP_CONTROLLER_WHOIS_ENABLED` is set to ``True`` but the
+    required environment variables are not defined, an
+    ``ImproperlyConfigured`` exception will be raised.
+
+    Both :ref:`OPENWISP_CONTROLLER_WHOIS_GEOIP_ACCOUNT` and
+    :ref:`OPENWISP_CONTROLLER_WHOIS_GEOIP_KEY` must be set to allow
+    per-organization control of the WHOIS Lookup feature. If either is
+    missing, the feature will be globally disabled and unavailable in the
+    admin interface.
+
+The WHOIS Lookup feature retrieves information about the last public IP
+address reported by a device to assist with device identification and
+management.
+
+The fetched details include:
+
+- ASN (Autonomous System Number)
+- CIDR block
+- Physical address
+- Timezone
+- Name of the organization associated with the ASN
```

Comment:
Let's summarize when and how this feature is triggered: in short, we should summarize an overview of the logic introduced in this PR, eg: triggered only when the last_ip is a public IP and it has either changed or there was no prior whois info available. We should summarize other critical info like how is it automatically updated, caching expirations and so on, only critical overview info.